src/en/proposals/emogarbage-celestial-war-gamemode.md

```diff
@@ -264,7 +264,7 @@ Additionally spawns violent tendrils that move through the underbrush and attack
 _A long, floating cloak with only an upper torso. The base of their torso simply ends in a long spinal cord._
 
 Safrinni is the god of thievery and of rogues.
-Their divinity is raised through theft, like items off of unsuspecting players 
+Their divinity is raised through stealing items off of unsuspecting players.
```

Comment:
this will get farmed to shit "please let me take an item off you 30 times"

Reply:
kid named separating implementation from concept. you can simply add in preventative measures that make this non-viable.